src/Compilers/CSharp/Portable/Symbols/Source/LocalFunctionSymbol.cs

```diff
@@ -348,6 +364,11 @@ private ImmutableArray<TypeParameterSymbol> MakeTypeParameters(DiagnosticBag dia
                 var location = identifier.GetLocation();
                 var name = identifier.ValueText;
 
+                foreach (var attr in parameter.AttributeLists)
+                {
+                    diagnostics.Add(ErrorCode.ERR_AttributesInLocalFuncDecl, attr.Location);
+                }
```

Comment:
Consider extracting a helper method for the loop, for use here and in `ComputeParameters`.